### PR TITLE
[minecraft-server] New module

### PR DESCRIPTION
**Pull request by kvaps**
## Description

This PR adds [Minecraft](https://www.minecraft.net) server module to the Deckhouse Kubernetes platform

## Why do we need it, and what problem does it solve?

Minecraft is a great collaboration platform. It allows you and your co-workers to work together on moving cubes at work and holding meetings.
It also allows you to make great charts out of sheep's wool and run tests with red stone.

This is a first step before implementing [KubeCraftAdmin](https://eric-jadi.medium.com/minecraft-as-a-k8s-admin-tool-cf16f890de42) for managing Kubernetes cluster.

## Changelog entries

```changes
section: minecraft-server
type: feature
summary: "New module: minecraft-server"
impact_level: high
impact: |
  Minecraft is a great collaboration platform. It allows you and your co-workers to work together on moving cubes at work and holding meetings.
  It also allows you to make great charts out of sheep's wool and run tests with red stone.
```

### CODE REVIEW

File: modules/900-minecraft-server/docs/USAGE_RU.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+title: "Модуль minecraft-server: примеры конфигурации"
+---
+
+### Настройка клиента
+
+1. Запустите клиент [Minecraft](https://www.minecraft.net/ru-ru)
+1. Выберите режим игры "Multiplayer"
+1. Нажимте "Direct Connect"
+1. В поле "Server Address" вбейте адрес вашего сервера
+1. Нажимте "Join Server"
```
Comment (z9r5) [inline modules/900-minecraft-server/docs/USAGE_RU.md:11]:
```suggestion
1. Запустите клиент [Minecraft](https://www.minecraft.net/ru-ru)
1. Выберите режим игры "Multiplayer"
1. Нажмите "Direct Connect"
1. В поле "Server Address" вбейте адрес вашего сервера
1. Нажмите "Join Server"
```

Comment (kvaps) [reply modules/900-minecraft-server/docs/USAGE_RU.md]: thanks, fixed!

File: modules/900-minecraft-server/docs/CONFIGURATION.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+---
+title: "The minecraft-server module: configuration"
+---
+
+This module is **disabled** by default. To enable it, add the following lines to the `deckhouse` ConfigMap:
```
Comment (deckhouse-BOaTswain) [inline modules/900-minecraft-server/docs/CONFIGURATION.md:5]: Why it is disabled? I think it should be a default UI.

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've started the workflow for label 'deploy/web/stage'...

File: modules/900-minecraft-server/docs/CONFIGURATION.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+```
+data:
+  minecraftServerEnabled: "true"
+```
+
+## Parameters
+
+<!-- SCHEMA -->
```
Comment (z9r5) [inline modules/900-minecraft-server/docs/CONFIGURATION.md:13]:
```suggestion
This module does not require any configuration and is **disabled** by default. To enable it, add the following lines to the `deckhouse` ConfigMap:
```
data:
  minecraftServerEnabled: "true"
```

```

File: modules/900-minecraft-server/docs/CONFIGURATION.md (hunk 2 above)
Comment (z9r5) [inline modules/900-minecraft-server/docs/CONFIGURATION.md:5]:
```suggestion
This module does not require any configuration and is **disabled** by default. To enable it, add the following lines to the `deckhouse` ConfigMap:
```

File: modules/900-minecraft-server/docs/CONFIGURATION.md (hunk 3 above)
Comment (z9r5) [inline modules/900-minecraft-server/docs/CONFIGURATION.md:13]:
```suggestion
```

File: modules/900-minecraft-server/docs/CONFIGURATION_RU.md [hunk 4]
```diff
@@ -0,0 +1,13 @@
+---
+title: "Модуль minecraft-server: настройки"
+---
+
+Модуль по умолчанию **выключен**. Для включения добавьте в ConfigMap `deckhouse`:
```
Comment (z9r5) [inline modules/900-minecraft-server/docs/CONFIGURATION_RU.md:5]:
```suggestion
Модуль не требует настройки и **выключен** по умолчанию. Для включения добавьте в ConfigMap `deckhouse`:
```

File: modules/900-minecraft-server/docs/CONFIGURATION_RU.md [hunk 5]
```diff
@@ -0,0 +1,13 @@
...
+```
+data:
+  minecraftServerEnabled: "true"
+```
+
+## Параметры
+
+<!-- SCHEMA -->
```
Comment (z9r5) [inline modules/900-minecraft-server/docs/CONFIGURATION_RU.md:13]:
```suggestion
```

**Review by z9r5 — APPROVED** [review]
Added some suggestions

**Comment by kvaps** [comment]
We decided that most of Deckhouse users are 12+ years old and the minecraft module is not suitable for us

**Pull request closed without merging**